crates/typst-library/src/foundations/datetime.rs

```diff
@@ -274,7 +274,29 @@ impl Datetime {
         /// The second of the datetime.
         #[named]
         second: Option<u8>,
-    ) -> StrResult<Datetime> {
+    ) -> HintedStrResult<Datetime> {
+        fn format_missing_fields(fields: Vec<&str>) -> String {
```

Comment:
If we stay with the `Vec` approach, I suppose it is nicer if this is a `&[&str]`.